const protected_route = (req, res, next) => {
    try {
        
    } catch (error) {
        console.log("There was a problem with the ")
    }
}

export default protected_route